docs/get-started/lotus/send-and-receive-fil.md

```diff
@@ -44,16 +44,43 @@ Remember that your will only see the latest balance when your daemon is fully sy
 
 ## Sending FIL
 
-Sending some FIL can be achieved by running:
+Sending some FIL from default wallet can be achieved by running:
 
 ```bash
 lotus wallet send <address> <amount>
 ```
+To get default wallet address:
+
+```bash
+lotus wallet default
+```
+
+To set default wallet address to a different address:
+
+```bash
+lotus wallet set-default <address>
+```
+
+To send some FIL from a specific wallet can be achieved by running:
+
+```bash
+lotus send --from=<sender address> <target address> <amount>
+```
 
 :::tip
 Make sure to check `lotus wallet send --help` for advanced options.
 :::
 
+## Withdraw FIL from the miner actor
+
+To withdraw available balance from miner actor to worker address, run
+
+```bash
+lotus-miner actor withdraw <amount>
+```
+
+> Note: Make sure that your worker address has sufficient balance to pay for the withdraw message's gas fee.
```

Comment:
Can you provide a little more guidance here, like typical message gas fee costs or how to check if your message has insufficient funds?

Reply:
We need to find a better place for this, as this is intended for miners and the get-started section is not for that. Might make a specific guide for miner-wallet management that includes this and https://docs.filecoin.io/mine/lotus/separate-address-window-post/

In the meantime we can leave it out of this PR to merge the other part.